src/mcts/search.h

```diff
@@ -196,6 +197,22 @@ class Search {
   friend class SearchWorker;
 };
 
+// Helper to score and sort terminals for preferred moves.
+class Search::TerminalScore {
```

Comment:
Hmmm there should be better name than TerminalScore..
Long name would be something like `Terminal Edge Ordering Wrapper`.

`TerminalSortingKey`? Meh..

Reply:
Removed the class.